app/controllers/organizations_controller.rb

```diff
@@ -102,15 +102,23 @@ def setup_organization
     end
   end
 
+  # rubocop:disable MethodLength
   def search
-    orgs_found = current_user.organizations.order(:id).where("title LIKE ?", "%#{params[:query]}%")
+    @organization = current_user
```

Comment:
Could you rename this to `@organizations`? That comment might have been lost with the refactoring comment :)